resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -61,15 +61,15 @@ class KubernetesSuite extends SparkFunSuite
   protected var appLocator: String = _
 
   // Default memory limit is 1024M + 384M (minimum overhead constant)
-  private val baseMemory = s"${1024 + 384}Mi"
```

Comment:
Hi, @ScrapCodes . Do we need this change in this PR?

Reply:
Hi Dongjoon, Yes, it is part of updating the client version. Tests won't pass otherwise.

Reply:
To be precise, in 4.7.1 Quantity constructor tries to parse the amount into format and the amount. [link](https://github.com/fabric8io/kubernetes-client/blob/v4.7.1/kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/Quantity.java#L69)
And in 4.6.4, It does not. [link](https://github.com/fabric8io/kubernetes-client/blob/v4.6.4/kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/Quantity.java#L65)
More details here : https://github.com/fabric8io/kubernetes-client/commit/c370d2628aa5da6b7217486373e16a35d7013df2

Reply:
Hi @dongjoon-hyun, a gentle ping.

Reply:
Thanks!